src/SSVOpenHexagon/Core/HGUpdate.cpp

```diff
@@ -181,8 +183,7 @@ void HexagonGame::updateTimeStop(FT mFT)
 {
     if(status.timeStop <= 0)
     {
-        status.currentTime += ssvu::getFTToSeconds(mFT);
-        status.incrementTime += ssvu::getFTToSeconds(mFT);
+        status.updateTime();
```

Comment:
Check to make sure this new implementation doesn't break u_haltTime, which will use status.timeStop